src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/CcdCaseRetriever.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+@Service
+public class CcdCaseRetriever {
```

Comment:
Doesn't seem to give any value currently?
Because it has retriever in it its only ever going to have one method in it
and the method is quite simple anyway?

Reply:
The responsibility for this object is to adapt the information retrieved from the authenticator to the call made to the CaseAPI. 
It will in effect l protect the other classes from the slight complexities with the case api and changes it might introduce. We could inject the authenticator here and it will encapsulate the whole effort to retrieve the case for the jurisdiction using the jurisdiction user.

Reply:
I would think about dropping retriever, its not really doing any work currently, and could be used for other CCD operations?